queries/vouch_registry.sql

```diff
@@ -43,6 +43,7 @@ invalidations as (
         on pool = "bondingPool"
         and sender = initial_funder
   where evt_block_number <= (select * from last_block_before_timestamp)
+    and evt_block_number < 15022969 -- block of the invalidation transaction
```

Comment:
Transactions involving excluded invalidations:

https://etherscan.io/tx/0xce690e766657b384c182f05484fc802e297f101d3b7e7288339ecea506090dfb#eventlog

https://etherscan.io/tx/0x14f9038e5f9cd7834318fe1b0dcb08d6778363f6f8ffe61b6e6f1611fca81f77#eventlog

Note that the second one happened in a later block so we use the earliest block number.

Reply:
I think you want the second transaction block number. The first transaction was used to invalidate solvers that we didn't end up using (there was a bug in the PK generation).

Reply:
Oh yeah, that makes sense, the results would be the same (since we actually never used - or even funded - the solver keys that got unvouched in the first transaction).